quantum/secure.c

```diff
@@ -85,3 +88,16 @@ void secure_task(void) {
     }
 #endif
 }
+
+#if defined(SECURE_ENABLE)
```

Comment:
Preprocessor check shouldn't be required -- this file is already conditionally added just through `common_features.mk`?